### ISSUE TITLE
[NRCE] Move all Meta fields from Input node to Actor node

### ISSUE THREAD

**Issue by spants** [enocean-js/enocean-js#146]
Can the Actor node do the check that it is a valid packet and generate the meta/data?.
Although, for instance, RSSI is shown in the configuration of the Actor node for a device, it is not exposed to the following flow

**Comment by Holger-Will** [comment]
> Can the Actor node do the check that it is a valid packet and generate the meta/data?
i would prefer not to...

but if you add enocean-js to your function gloabel context in your settings.js file:

```
functionGlobalContext: {
        Enocean: require('enocean-js'),
}
```
you can perform the test and extraction of the meta in a function node like this:

```
const Enocean = global.get('Enocean')
var tel = Enocean.RadioERP1.from("55000d0701fdd4a0ff33000901d2019167fa0001ffffffff4700ec")
...
```

**Comment by Holger-Will** [comment]
... let me think about this for a bit...

**Comment by Holger-Will** [comment]
fixed with #132
